src/w2form.js

```diff
@@ -755,6 +755,7 @@ class w2form extends w2base {
             }
             if (field.options && field.hidden !== true && field.options.minLength > 0
                     && ['enum', 'list', 'combo'].indexOf(field.type) == -1 // since minLength is used there too
+                    && val != null && val != ''
```

Comment:
Well, since val is already calcu;ated from `getField(field.field)`, I think we could just do this `&& val != null && val.length < field.options.minLength` and it should cover all casses